src/js/ia/IADevPipeline.js

```diff
@@ -16,6 +16,9 @@
                 var iadp;
                 iadp = Handlebars.templates.dev_pipeline(data.dev_milestones);
                 $("#dev_pipeline").html(iadp);
+
+                // 100% width
+                $(".site-main > .content-wrap").first().removeClass("content-wrap").addClass("wrap-pipeline");
```

Comment:
That's what we do in the single page as well, otherwise we couldn't have that blue band with 100% width.

I thought we agreed we need more space on the pipeline, at least for now that we only use the grid view.

How/where do you want to note this fact?